cds_ils/series/api.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019-2020 CERN.
+#
+# CDS-ILS is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+"""CDS-ILS series API."""
+
+from invenio_app_ils.series.api import Series as ILSSeries
+from invenio_pidstore.resolver import Resolver
+
+DOCUMENT_LEGACY_PID_TYPE = "lrecid"
+
+
+class Series(ILSSeries):
+    """CDS-Ils document class."""
+
+    @classmethod
+    def get_record_by_legacy_recid(cls, pid_value):
+        """Get ils record by pid value and pid type."""
+        resolver = Resolver(
+            pid_type=DOCUMENT_LEGACY_PID_TYPE,
```

Comment:
```suggestion
            pid_type=SERIES_LEGACY_PID_TYPE,
```
or 
```suggestion
            pid_type=RECORD_LEGACY_PID_TYPE,
```
in general it would be better with `RECORD_LEGACY_PID_TYPE`, and maybe to move it into the config, since it is used by both Document and Series